docs/src/quickstart.md

```diff
@@ -85,16 +94,25 @@ The algorithm solves at each step the linear equation:
 Hence, the algorithm requires the following elements:
 
 - an initial position $x_0$
-- a function to evaluate the Jacobian $\nabla_x g_k$
 - a function to solve efficiently the linear system $(\nabla_x g_k) x_{k+1} = g(x_k, u)$
+- a function to evaluate the Jacobian $\nabla_x g_k$
 
 that translate to the Julia code:
 ```julia-repl
-julia> physical_state = get(polar, PhysicalState())
-julia> jx = ExaPF.init_ad_factory(polar, physical_state)
-julia> linear_solver = DirectSolver()
+julia> physical_state = get(polar, ExaPF.PhysicalState())
+julia> ExaPF.init_buffer!(polar, physical_state) # populate values inside buffer
+julia> linear_solver = LS.DirectSolver()
 
 ```
+
+We build a Jacobian object storing all structures needed by
+the AutoDiff backend:
+```julia-repl
+julia> jx = AutoDiff.Jacobian(polar, ExaPF.power_balance, State())
+A AutoDiff Jacobian for power_balance (w.r.t. State())
+Number of Jacobian colors: 25
+```
```

Comment:
I see you use `julia-repl` here. I think we should separate the output from the code though for the same reasons than above.